library/ssl_tls.c

```diff
@@ -109,6 +109,39 @@ static void ssl_update_in_pointers( mbedtls_ssl_context *ssl,
 
 #if defined(MBEDTLS_SSL_PROTO_DTLS)
 
+/* Top-level Connection ID API */
```

Comment:
Found by the CI (test-ref-configs): the definition should be guarded by the same `#ifdef`s as the declaration.

Reply:
Done.